### PR TITLE
Update Solr training

### PR DESCRIPTION
**Pull request by reinhardt**
The part in the setup where I download half a dozen files with curl is not particularly pretty, but I can't think of a much nicer way short of cloning the whole repository (which seems like overkill). Suggestions welcome, but it at least it works now.

There are some outdated bits left. I hope to find some time to update them soonish.

Refs #475

### CODE REVIEW

**Comment by reinhardt** [comment]
@balavec, if you want you can try the "Setup" part already. I'm still polishing some of the rest.

**Comment by reinhardt** [comment]
There are still some outdated bits, but I don't know when I'll get around to continue working on this. Let's merge it, at least the setup part should work now.

File: solr/setup.rst [hunk 1]
```diff
@@ -27,23 +31,28 @@ Create a buildout (*buildout.cfg*) which installs both requirements
...
+    collective.solr = 8.0.0a5
+    kitconcept.recipe.solr = 1.0.0a5
 
 
 Run buildout:
 
 .. code-block:: console
 
-   python2.7 bootstrap-buildout.py
+   python bootstrap-buildout.py
```
Comment (tomgross) [inline solr/setup.rst]: Is this still state of the art, or virtualenv / pip?

Comment (reinhardt) [reply solr/setup.rst]: Ah, you're right, that should probably be updated as well in the next round. Thanks!

**Pull request merged**